cmd/notifyStale.go

```diff
@@ -83,10 +83,11 @@ var notifyStaleCmd = &cobra.Command{
 				}
 
 				if shouldSendMessage {
-					message := fmt.Sprintf("The following pull request is waiting for approval for CI checks to run: %s", pr.URL)
+					message := fmt.Sprintf("The following pull request has activity from a Chia team member in the last 7 days: %s", pr.URL)
```

Comment:
Should this maybe be the opposite? No activity? Or am I thinking of a different alert...

Reply:
You are correct